Fix failing tests when webpack mode="production"

These tests fail when `mode: "production"` in `webpack.config.js`:

```
1) should not have 'grayscale' class if movie has been watched
     MCUMovie
     AssertionError: expected [ 'movie', 'grayscale' ] to not include 'grayscale'
    at Context.<anonymous> (webpack:///test/MCUMovie.spec.ts:46:41 <- assets/js/test/MCUMovie.spec.bundle.js:580:1011511)

2) should appear if both loaded and available to show
     MCUMovie
     AssertionError: expected 'none' to not equal 'none'
    at Context.<anonymous> (webpack:///test/MCUMovie.spec.ts:59:53 <- assets/js/test/MCUMovie.spec.bundle.js:580:1011701)

3) should populate the dropdown with options
     MCUSelector
     AssertionError: expected 0 to equal 2
    at Context.<anonymous> (webpack:///test/MCUSelector.spec.ts:44:36 <- assets/js/test/MCUSelector.spec.bundle.js:588:2970)

4) should invoke save operation for correct ordering preference when selected
     MCUSelector
     AssertionError: did not invoke save for the correct ordering: expected undefined to equal 'test2'
    at Context.<anonymous> (webpack:///test/MCUSelector.spec.ts:63:36 <- assets/js/test/MCUSelector.spec.bundle.js:588:3348)
    at s (webpack:///node_modules/sinon-test/dist/sinon-test.js:146 <- assets/js/test/MCUSelector.spec.bundle.js:588:1293)
    at Context.<anonymous> (webpack:///node_modules/sinon-test/dist/sinon-test.js:166 <- assets/js/test/MCUSelector.spec.bundle.js:588:1578)
```

Investigate and fix.

--- a/test/MCUMovie.spec.ts
+++ b/test/MCUMovie.spec.ts
@@ -26,33 +26,39 @@ describe("MCUMovie", () => {
         expect(imgElem.classes()).to.contain("movie");
     });
 
-    it("should have 'grayscale' class if movie not yet watched", () => {
+    it("should have 'grayscale' class if movie not yet watched", async () => {
         wrapper.setData({
             watched: false
         });
+
+        await Vue.nextTick();
 
         let imgElem : Wrapper<Vue> = wrapper.find("img");
 
         expect(imgElem.classes()).to.contain("grayscale");
     });
 
-    it("should not have 'grayscale' class if movie has been watched", () => {
+    it("should not have 'grayscale' class if movie has been watched", async () => {
         wrapper.setData({
             watched: true
         });
+
+        await Vue.nextTick();
 
         let imgElem: Wrapper<Vue> = wrapper.find("img");
 
         expect(imgElem.classes()).to.not.contain("grayscale");
     });
 
-    it("should appear if both loaded and available to show", () => {
+    it("should appear if both loaded and available to show", async () => {
         wrapper.setData({
             loaded: true
         });
         wrapper.setProps({
             show: true
         });
+
+        await Vue.nextTick();
 
         let imgElem: Wrapper<Vue> = wrapper.find("img");
         
@@ -80,7 +86,7 @@ describe("MCUMovie", () => {
                 watched: false
             });
             
-            wrapper.vm.toggleWatch(); // TODO find a way to call this method in a way that TypeScript will understand
+            wrapper.vm.toggleWatch();
 
             expect(wrapper.vm.watched).to.equal(true);
         });
@@ -92,7 +98,7 @@ describe("MCUMovie", () => {
                 watched: true
             });
             
-            wrapper.vm.toggleWatch(); // TODO find a way to call this method in a way that TypeScript will understand
+            wrapper.vm.toggleWatch();
 
             expect(wrapper.vm.watched).to.equal(false);
         });

--- a/test/MCUSelector.spec.ts
+++ b/test/MCUSelector.spec.ts
@@ -17,7 +17,7 @@ describe("MCUSelector", () => {
 
     const TEST_OPTIONS : string[] = ["test1", "test2"];
 
-    beforeEach(test(function() {
+    beforeEach(test(async function() {
         movieModelStub = new MovieSeries("test", [], {}, null);
         movieModelStub.getCurrentOrderingName = this.stub().returns("");
         movieModelStub.verifyOrdering = this.stub().returns(true);

